### ISSUE TITLE
[UX] Include the "+ Add condition" link/button and the "Remove"/"Configure" links in the "Conditions" fieldset of the 

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1025]
Basically replicate the same fieldset (w/o the help text perhaps) from the dialog inside the `.layout-editor-block-content` div:

![conditions_fieldset](https://cloud.githubusercontent.com/assets/2423362/8270745/fb05fb62-1833-11e5-80fa-5f5c9b57fab6.png)
- It would not complicate the UI since that fieldset is collapsed by default.
- It would save the user a couple of clicks if they wanted to either add a condition or edit/remove any existing ones.

**Comment by docwilmot** [comment]
@klonos why not a META for layout/block improvements. I'm finding a few technical issues as well.

**Comment by docwilmot** [comment]
https://github.com/backdrop/backdrop-issues/issues/345?

**Comment by klonos** [comment]
I just edited that META's summary to include all the recent issues I've created against the Layouts/Blocks UI.

**Comment by docwilmot** [comment]
Looks like layouts needs lots of love.

**Comment by klonos** [comment]
Indeed. Still one of the best features of Backdrop. Just needs to be polished.

**Comment by quicksketch** [comment]
This sounds like a good idea to me. However getting it right could be a little tricky. Right now adding/configuring conditionals doesn't save anything right away. You have to finish saving the block in order to update the underlying layout configuration.

Right now the "Conditions" fieldset on a block is also only shown if there are existing conditions. If we're going to add an "Add condition" link within this fieldset, does that mean we will always show the conditional fieldset even if there are no conditionals?

**Comment by klonos** [comment]
> ...does that mean we will always show the conditional fieldset even if there are no conditionals?

Hmm, I haven't thought of that. I'm leaning towards rendering another "+ Add condition" link outside the fieldset and hiding the fieldset when no conditions are yet defined. When there are conditions, we could hide that sole "+ Add condition" link and show the fieldset instead along with its "+ Add condition" link and the list of conditions.

I mean, there is no point to have the used "dig" inside a fieldset in order to access the "+ Add condition" link. Unnecessary click. This would also make it clear in the layout overview as to which blocks have conditions and which do not.

What do you think?

**Comment by docwilmot** [comment]
I'm thinking that the "Remove" link *may* be OK on the draggable block. But I'm inclined to think that this, plus the  "Add condition" and "configure" links should be accessed through the dialog as usual. Reason being there are other actions on block configure (Style settings, Menu styles, etc), and there is no reason why the condition links should get special treatment. If we're going to put the "Add condition" link there, will we also do the "Add style", "Configure menu" etc etc? Don't think so.

**Comment by klonos** [comment]
...yes, fair enough, but we are already special-casing the visibility conditions by listing them there :wink:

**Comment by docwilmot** [comment]
Almost started this but realized its a bit more than I thought. The "Remove" and "Configure" links in the block config form are actually form buttons, (made to look like links using JQuery) defined in `layout.admin.php`. While the Conditions list in the draggable blocks is just a simple `theme_item_links()` defined in `layout_renderer_standard.php`. 

We'd have to change the simple list to a form on the draggable. Perhaps doable but not sure its worth it. Let @quicksketch decide?
